src/components/VoiceCompair/VoiceCompair.js

```diff
@@ -38,6 +38,18 @@ const VoiceCompair = props => {
   const [tamilRecordedAudio, setTamilRecordedAudio] = useState('');
   const [tamilRecordedText, setTamilRecordedText] = useState('');
 
+  const [asr_language_code, set_asr_language_code] = useState('ai4bharat/whisper-medium-en--gpu--t4');
+
+  useEffect(() => {
+	if (lang_code === 'hi')
```

Comment:
Need to add switch condition here and set default to `en`

Reply:
Fixed.